spec/lib/billy/cache_spec.rb

```diff
@@ -78,7 +78,7 @@
       end
 
       it 'More specifically, the cache keys should be identical for the 2 analytics urls' do
-        identical_cache_key = 'post_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
+        identical_cache_key = 'post_www.example-analytics.com_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
```

Comment:
I'm still concerned about why this cache key changed in this PR.  Would cause a lot of breakages for existing users if this is different.